### PR TITLE
few minor doc improvements

### PR DESCRIPTION
**Pull request by rfourquet**
Not all changes are guaranteed to be improvements!!

### CODE REVIEW

File: src/REPL/command_declarations.jl [hunk 1]
```diff
@@ -300,10 +305,10 @@ The `startup.jl` file is disabled during precompilation unless julia is started
 
 Show the status of the current environment. By default, the full contents of
 the project file is summarized, showing what version each package is on and
-how it has changed since the last git commit (if in a git repo), as well as
+how it has changed since the last git commit (if in a git repository), as well as
 any changes to manifest packages not already listed. In `--project` mode, the
 status of the project file is summarized. In `--manifest` mode the output also
-includes the dependencies of explicitly added packages. If there are any
+includes the dependencies of implicitly added packages. If there are any
```
Comment (fredrikekre) [inline src/REPL/command_declarations.jl]: same here

File: docs/src/managing-packages.md [hunk 2]
```diff
@@ -36,7 +36,7 @@ you have added yourself, in this case, `Example`:
   [7876af07] Example v0.5.1
 ```
 
-The manifest status, in addition, includes the dependencies of explicitly added packages.
+The manifest status, in addition, includes the dependencies of implicitly added packages:
```
[the remark this answers is not in the document]
Comment (rfourquet) [reply docs/src/managing-packages.md]: Oh, I got confused, I read that as "The manifest status, in addition, includes the *status* of implicitly added packages". But actually, I observe that the manifest status shows the _recursive_ dependencies of explicitly added packages, so we could say "includes the dependencies of explicitly or implicitly added packages", which amounts to saying "includes the recursive dependencies of added packages". Is that last formulation fine for you?

[the remark this answers is not in the document]
Comment (fredrikekre) [reply docs/src/managing-packages.md]: Yea, maybe just write something like " The manifest status includes the status of all packages in the environment, including recursive dependencies."

**Comment by rfourquet** [comment]
> Please squash to one comit before merging.

Ok, done. Can't this be done on the github interface (or with bors) at merge time?

**Comment by fredrikekre** [comment]
bors can't, unfortunately.

bors r+

**Pull request merged**